draft-ietf-quic-transport.md

```diff
@@ -624,10 +720,11 @@ server, it MUST discard any packet it receives with a different Source
 Connection ID.
 
 
-#### Tokens
+### Tokens
 
-If the client has a suitable token available from a previous connection, it
-SHOULD populate the Token field.
+If the client has an unused token that it received in a NEW_TOKEN frame on a
+previous connection to what it believes to be the same server, it includes that
+value in the Token field of its Initial packet.
```

Comment:
This seems oddly non-normative.

Also, I notice you have removed "suitable" but if you think you changed your IP then you probably shouldn;t.

Reply:
The point is that it can't really be normative.  You can't say MUST for sure, and frankly the whole normative thing is not valuable here: there is no interoperability end served by including a token.

"suitable" wasn't defined, but I think that it just highlighted a deficiency, so I've add text on linkability.

Reply:
We have plenty of MUST requirements that don't serve interop. But SHOULD would be fine as well. Writing it as "it will" is just confusing.